project-one/application.py

```diff
@@ -93,10 +93,20 @@ def book_search():
 
 
 # the book details page
-@app.route("/books/<isbn>")
+@app.route("/books/<isbn>/<title>")
 def book_details(isbn,title):
     print(isbn, title)
-    # return render_template('book.html', )
+    book_data = Books.query.filter_by(isbn = isbn).first()
+    reviews_list = Reviews.query.filter_by(book_isbn = isbn).all()
+    # print(reviews_list)
```

Comment:
remove commented code

Reply:
Okay bro. I will make the changes.

Reply:
I have made the changes. please review it. @ashishk144